pom.xml

```diff
@@ -154,7 +164,34 @@
                     </schemas>
                 </configuration>
             </plugin>
-
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-checkstyle-plugin</artifactId>
+                <version>2.17</version>
+                <executions>
+                    <execution>
+                        <id>checkstyle</id>
+                        <phase>validate</phase>
+                        <goals>
+                            <goal>check</goal>
+                        </goals>
+                        <configuration>
+                            <configLocation>google_checks.xml</configLocation>
+                            <encoding>UTF-8</encoding>
+                            <consoleOutput>true</consoleOutput>
+                            <failsOnError>true</failsOnError>
+                            <violationSeverity>warning</violationSeverity>
+                            <failOnViolation>true</failOnViolation>
+                            <suppressionsLocation>
+                                checkstyle-suppressions.xml
+                            </suppressionsLocation>
+                            <suppressionsFileExpression>
```

Comment:
I'm not sure you need this; I think it is related to configuring checkstyle ignored filed via a properties file. If you have `suppressionsLocation`, you might be good.

Reply:
I'll take a look at this tomorrow morning.

Reply:
Outside of the pom file everything else is fixing linter errors.

Reply:
I didn't get to looking at this; probably tomorrow.